src/sagas/nanoContract.js

```diff
@@ -158,6 +158,16 @@ export function* registerNanoContract({ payload }) {
   // emit action NANOCONTRACT_REGISTER_SUCCESS with feedback to user
   yield put(nanoContractRegisterSuccess({ entryKey: ncId, entryValue: nc, hasFeedback: true }));
 }
+/**
+ * Effect invoked by safeEffect if an unexpected error occurs.
+ *
+ * @param {Object} error The error captured.
+ */
+function* registerNanoContractOnError(error) {
+  log.error('Unexpected error while registering Nano Contract.', error);
+  yield put(nanoContractRegisterFailure(failureMessage.blueprintInfoFailure));
```

Comment:
question: Why use the `blueprintInfo` failure when the error was on a registration event?

Reply:
Fixed: 4fb72eb794ed26372460d5700c09d357284141bb